OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -104,13 +120,16 @@ public ConnectionFailedLogic(Widget widget, ModData modData, OrderManager orderM
 				onRetry(pass);
 			};
 
-			widget.Get<LabelWidget>("CONNECTING_DESC").GetText = () => $"Could not connect to {connection.Target}";
+			var connectingDescText = modData.Translation.GetString(CouldNotConnectToTarget, Translation.Arguments("target", connection.Target));
+			widget.Get<LabelWidget>("CONNECTING_DESC").GetText = () => connectingDescText;
 
 			var connectionError = widget.Get<LabelWidget>("CONNECTION_ERROR");
-			connectionError.GetText = () => modData.Translation.GetString(orderManager.ServerError) ?? connection.ErrorMessage ?? "Unknown error";
+			var connectionErrorText = modData.Translation.GetString(orderManager.ServerError) ?? connection.ErrorMessage ?? modData.Translation.GetString(UnknownError);
```

Comment:
Does `orderManager.ServerError` need to support arguments? (or will it in the future?)

Reply:
I can't answer this and it seems unrelated to this pull request.